### ISSUE TITLE
pymessenger multi-threading

### ISSUE THREAD

**Issue by acondoriml** [davidchua/pymessenger#10]
[PYTHON - MESSENGER] Hi! How can I serve multiple clients using  multi-threading? Thank you !

**Comment by tbsf** [comment]
Are you asking if this module is thread-safe?

**Comment by GoelBiju** [comment]
I think @acondoriml is referring to handling multiple connections at once. This might not be possible to do.

**Comment by acondoriml** [comment]
@GoelBiju Yes, I want handling multiple connections at once, I try using gunircon, apache, flask, I set up several ways and just handing one connections at once, is a pain. I can understand why it doesn`t work

**Comment by davidlowjw** [comment]
Anyone has solution for this? Hi @acondoriml Did you manage to figure it out?

**Comment by acondoriml** [comment]
@davidlowjw I can't find a solution, it's a shame because the library works great!

**Comment by neoscott** [comment]
If you use Flask with Gunicorn, uwgsi and set more than 1 worker you will have the number of workers to reply multiple clients.
If you are using Flask without any upper layer like gunicorn,... you only are going to handle one request per time.
